src/app/components/team/SmoochBot/SmoochBotConfig.js

```diff
@@ -151,14 +150,25 @@ const SmoochBotConfig = (props) => {
 
   return (
     <React.Fragment>
-      <Tabs value={currentTab} onChange={handleChangeTab} variant="fullWidth">
-        <Tab label={<FormattedMessage id="smoochBot.designYourBot" defaultMessage="Design your bot" description="Title of tipline settings page" />} />
-        { userRole === 'admin' ?
-          <Tab label={<FormattedMessage id="smoochBot.settings" defaultMessage="Settings" description="Tab label to click to see the settings area" />} />
-          : null
-        }
-      </Tabs>
-      { currentTab === 0 ?
+      { userRole === 'admin' ?
```

Comment:
moved the logic here because if you are not an admin, then there is only 1 tipline setting area available to you, so you don't need to see a "Tab" with 1 choice